tasks/pipeline.py

```diff
@@ -23,24 +23,13 @@
 ALLOWED_REPO_BRANCHES = {"stable", "beta", "nightly", "none"}
 
 
-@task
-def trigger(_, git_ref="master", release_version_6="nightly", release_version_7="nightly-a7", repo_branch="nightly"):
```

Comment:
To be less disruptive with people's workflow, should we keep this task (and just call `run` with `deploy=true` and print a notice saying it's deprecated?

Reply:
We can 👍